src/platform/forms-system/src/js/containers/FormPage.jsx

```diff
@@ -13,7 +14,11 @@ import { getNextPagePath, getPreviousPagePath } from '../routing';
 import { focusElement } from '../utilities/ui';
 
 function focusForm() {
-  focusElement('.nav-header');
+  if (environment.isProduction()) {
+    focusElement('.nav-header');
+  } else {
+    focusElement('.nav-header h2');
```

Comment:
as per the ticket, the h2 element needs to be selected. opted for using a nested selector rather than adding another class name.

Reply:
Why the isProduction check?

Reply:
I don't want it going out to prod until trevor tests it.